tests/transform_dialect/cuda/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+################################################################################
+# Autogenerated by build_tools/bazel_to_cmake/bazel_to_cmake.py from           #
+# tests/transform_dialect/cuda/BUILD                                           #
+#                                                                              #
+# Use iree_cmake_extra_content from iree/build_defs.oss.bzl to add arbitrary   #
+# CMake-only content.                                                          #
+#                                                                              #
+# To disable autogeneration for this file entirely, delete this header.        #
+################################################################################
+
+iree_add_all_subdirs()
+
+iree_lit_test_suite(
+  NAME
+    lit
+  SRCS
+    "reduction.mlir"
+    "transform_dialect_codegen_spec.mlir"
+    "transform_dialect_dispatch_spec.mlir"
+  TOOLS
+    FileCheck
+    iree-compile
+    iree-opt
+    iree-run-module
+  DATA
+    transform_dialect_codegen_spec.mlir
+    transform_dialect_dispatch_spec.mlir
+  LABELS
+    "noasan"
+    "nomsan"
+    "notsan"
+    "noubsan"
+    "requires-gpu-nvidia"
```

Comment:
This label on its own won't work together with `-DIREE_HAL_DRIVER_CUDA=OFF`. If someone builds with the CUDA target or driver disabled, these tests will still run, and they will fail. `iree_check_test` automatically excludes like this: 
https://github.com/iree-org/iree/blob/e3cbeac692b54f76ba26db1f7ee3fc303dfe6351/build_tools/cmake/iree_check_test.cmake#L227-L241

In lieu of that automatic filtering, you could manually exclude like what is done here: https://github.com/iree-org/iree/blob/e3cbeac692b54f76ba26db1f7ee3fc303dfe6351/samples/simple_embedding/CMakeLists.txt#L208-L209

that is generated by the verbose `iree_cmake_extra_content` in the BUILD file: https://github.com/iree-org/iree/blob/e3cbeac692b54f76ba26db1f7ee3fc303dfe6351/samples/simple_embedding/BUILD#L214-L222

(in this case the check would be for `IREE_HAL_DRIVER_CUDA` and `IREE_TARGET_BACKEND_CUDA`, ignore the `IREE_HOST_BINARY_ROOT` stuff since a lit test won't run when cross compiling anyways)